Dockerfile.rhoai

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM registry.access.redhat.com/ubi9/go-toolset:1.21 as builder
+
+WORKDIR /workspace
+
+RUN git config --global --add safe.directory /workspace
+
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY . .
+
+# Build
+RUN make build GO_BUILD_ENV='CGO_ENABLED=1 GOOS=linux'
+
+FROM registry.redhat.io/ubi9/ubi:latest
+
+WORKDIR /workspace
```

Comment:
We can get rid of the `workspace` directory in the runtime image and copy the binary at the root.

Reply:
Actually, l tried some ways to make it work in root directory itself.. but still lokking for how it can be done !

Reply:
I have made some changes again to use root directory for runtime image !